crates/re_viewer/src/ui/welcome_screen/python_quick_start.rs

```diff
@@ -0,0 +1,89 @@
+use re_viewer_context::{SystemCommand, SystemCommandSender};
+
+pub(super) fn python_quick_start(
+    command_sender: &re_viewer_context::CommandSender,
+) -> re_sdk::RecordingStreamResult<()> {
+    let (rec, storage) = re_sdk::RecordingStreamBuilder::new("Python Getting Started").memory()?;
+
+    rec.log(
+        "markdown",
+        &re_sdk::TextDocument::new(
+            r#"
+## Python Quick Start
+
+### Installing the Rerun SDK
+
+The Rerun SDK is available on [PyPI](https://pypi.org/) under the
+[`rerun-sdk`](https://pypi.org/project/rerun-sdk/) name. It can be installed like any other
+Python package:
+
+```sh
+pip3 install rerun-sdk
+```
+
+### Try out the viewer
+
+The Rerun SDK comes with a demo that can be used to try the viewer. You can send a demo recording
+to this viewer using the following command:
+
+```sh
+python3 -m rerun_sdk.demo --connect
```

Comment:
note that this won't work with the web-viewer

Reply:
Yes, that's what I meant by the "specialise Quick Start for a web-viewer-based workflow (which needs to be clarified)" TODO line.